tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -797,4 +797,18 @@ public String toString() {
     return sb.toString();
   }
 
+  public TiDAGRequest copy() {
+    try {
+      ByteArrayOutputStream baos = new ByteArrayOutputStream();
+      ObjectOutputStream oos = new ObjectOutputStream(baos);
+      oos.writeObject(this);
+      ByteArrayInputStream bais = new ByteArrayInputStream(baos.toByteArray());
+      ObjectInputStream ois = new ObjectInputStream(bais);
+      return ((TiDAGRequest) ois.readObject());
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
It is not advisable to do `printStackTrace` in production code. Some exception handling is recommended here.

Reply:
i will modify this function later.